src/Config/DatabasePrimitives/DatabaseObject.cs

```diff
@@ -186,6 +203,28 @@ public bool IsAnyColumnNullable(List<string> columnsToCheck)
 
         return null;
     }
+
+    /// <summary>
+    /// Method to get the system type for:
+    /// 1. column for table/view,
+    /// 2. parameter for stored procedure.
```

Comment:
nit: description related to stored procedure parameters can be removed if inapplicable here